odl/tomo/data/mrc2014.py

```diff
@@ -0,0 +1,197 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Specification and reader for the MRC2014 file format."""
+
+import numpy as np
+
+from odl.tomo.data.uncompr_bin import (
+    FileReaderUncompressedBinary, standardized_fields, spec_fields)
+
+
+__all__ = ('FileReaderMRC',)
+
+
+MRC_2014_SPEC = """
++---------+-------+---------+--------+-------------------------------+
+|Long word|Byte   |Data type|Name    |Description                    |
++=========+=======+=========+========+===============================+
+|1        |1-4    |Int32    |NX      |Number of columns              |
++---------+-------+---------+--------+-------------------------------+
+|2        |5-8    |Int32    |NY      |Number of rows                 |
++---------+-------+---------+--------+-------------------------------+
+|3        |9-12   |Int32    |NZ      |Number of sections             |
++---------+-------+---------+--------+-------------------------------+
+|4        |13-16  |Int32    |MODE    |Data type                      |
++---------+-------+---------+--------+-------------------------------+
+|...      |       |         |        |                               |
++---------+-------+---------+--------+-------------------------------+
+|8        |29-32  |Int32    |MX      |Number of intervals along X of |
+|         |       |         |        |the "unit cell"                |
++---------+-------+---------+--------+-------------------------------+
+|9        |33-36  |Int32    |MY      |Number of intervals along Y of |
+|         |       |         |        |the "unit cell"                |
++---------+-------+---------+--------+-------------------------------+
+|10       |37-40  |Int32    |MZ      |Number of intervals along Z of |
+|         |       |         |        |the "unit cell"                |
++---------+-------+---------+--------+-------------------------------+
+|11-13    |41-52  |Float32  |CELLA   |Cell dimension in angstroms    |
++---------+-------+---------+--------+-------------------------------+
+|...      |       |         |        |                               |
```

Comment:
To indicate we're skipping fields - not really needed but makes it a bit more clear.